[VDA-Reward-Contract] Updated to use ERC20 & Verification library

ITStar10 opened the pull request:
VDA-Reward-Contract:
	Removed RewardToken.sol(this is used for test only)
	Add dependency to the VDA-Verification-Base contract
	Test code updated

VDA-Verification-Base
	Minor updates on addTrustedSigner() function

All updates:
	Added UpdateHistory.md file to the all contracts.
	Updated package.json

ITStar10 committed:
[VDA-Reward-Contract] Updated to use ERC20 & Verification library

VDA-Reward-Contract:
	Removed RewardToken.sol(this is used for test only)
	Add dependency to the VDA-Verification-Base contract
	Test code updated

VDA-Verification-Base
	Minor updates on addTrustedSigner() function

All updates:
	Added UpdateHistory.md file to the all contracts.
	Updated package.json

ITStar10 committed:
Minor update : Removed comments

tahpot commented on VDA-Reward-Contract/test/RewardContract.ts:
```diff
@@ -4,40 +4,35 @@ import chaiAsPromised from "chai-as-promised";
+import { RewardToken } from "@verida/reward-token-contract/typechain-types";
 import EncryptionUtils from '@verida/encryption-utils'
 
+import { abi as TokenABI, bytecode as TokenByteCode } from "@verida/reward-token-contract/artifacts/contracts/RewardToken.sol/RewardToken.json";
```
This now needs to use our ERC20, not a reward token.

ITStar10 replied:
Updated

tahpot reviewed (changes requested):
Please use `CHANGELOG.md` filename instead of `UpdateHistory.md`.

ITStar10 committed:
Updates on review

- VDA-Reward-Contract
	Update test code to use the VeridaToken as a reward token

- All contract update
	Renamed all UpdateHistory.md to ChangeLog.md
